src/toolkits/object_detection/object_detector.cpp

```diff
@@ -224,26 +224,47 @@ void object_detector::init_options(
       1,
       std::numeric_limits<int>::max());
   options.create_integer_option(
-      /* name          */ "grid_height",
-      /* description   */
+      /* name             */ "grid_height",
+      /* description      */
       "Height of the grid of features computed for each image",
-      /* default_value */ 13,
-      /* lower_bound   */ 1,
-      /* upper_bound   */ std::numeric_limits<int>::max());
+      /* default_value    */ 13,
+      /* lower_bound      */ 1,
+      /* upper_bound      */ std::numeric_limits<int>::max());
   options.create_integer_option(
-      /* name          */ "grid_width",
-      /* description   */
+      /* name             */ "grid_width",
+      /* description      */
       "Width of the grid of features computed for each image",
-      /* default_value */ 13,
-      /* lower_bound   */ 1,
-      /* upper_bound   */ std::numeric_limits<int>::max());
+      /* default_value    */ 13,
+      /* lower_bound      */ 1,
+      /* upper_bound      */ std::numeric_limits<int>::max());
   options.create_integer_option(
       "random_seed",
       "Seed for random weight initialization and sampling during training",
       FLEX_UNDEFINED,
       std::numeric_limits<int>::min(),
       std::numeric_limits<int>::max());
-
+  options.create_categorical_option(
+      /* name              */ "annotation_scale",
+      /* description       */
+      "Defines annotations scale: pixel or normalized",
+      /* default_value     */ "pixel",
+      /* allowed_values    */ {flexible_type("pixel"), flexible_type("normalized")},
+      /* allowed_overwrite */ false);
+  options.create_categorical_option(
+      /* name              */ "image_origin",
```

Comment:
Could we make this "annotation_origin" instead?

Reply:
`image_origin` denotes the origin of the image (whether it is placed top-left or bottom-left), not the origin for annotation. So I'm thinking it may be unclear? What do you think

Reply:
I think it’s more clear to give all three options a common prefix, since all three define (collectively) the semantics of the annotations. Here, the “annotation_origin” defines the origin of the coordinate system used to specify the annotations. I don’t think it should be confused with “annotation_position”. Note that any single annotation is defined by a position and a size (like CGRect); I don’t think of the corner of a bounding box as the box’s origin.

Reply:
Hmmm, that's a good point. All three options collectively define the semantics of the annotations so it makes sense to give them the same prefix.

Reply:
FWIW, I could see "annotations_origin" being more clear than "annotation_origin". Then it's more obvious that "origin" is a property shared across all annotations. (Of course, then we would change all the prefixes to "annotations_".) Up to you if you think that's worth changing